src/main/java/com/exasol/bucketfs/ClusterConfigurationBucketFactory.java

```diff
@@ -48,6 +48,10 @@ private String getFullyQualifiedBucketName(final String serviceName, final Strin
         return serviceName + BucketConstants.PATH_SEPARATOR + bucketName;
     }
 
+    /**
```

Comment:
```suggestion
    /**
    Get bucket from chache.
    
```

Reply:
Did 'm everywhere I think. If you see some I missed or want to correct some feel free to change them or point them out.